extensions/blocks/podcast-player/components/with-error-boundary.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * External dependencies
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { Component } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+
+export default function withErrorBoundary( WrappedComponent ) {
```

Comment:
Side note: Nice idea to solve this issue with an Error Boundary. I'd assume `withErrorBoundary` could be useful in other parts of Jetpack as well, I am slightly surprised that this didn't exist as a shared util.

Reply:
I haven't made this one universal as the implementation is tied to our component, using our markup, the non-standard error callback etc… If there is need in other blocks, I'd be happy to make some sort of abstraction but since we only work on this one, I didn't do it just yet.

Reply:
Agreed, I think it makes sense to wait with abstractions until the need arises for use in more places. I was just noting that I would've thought this existed (but didn't).